kubernetes-services/work-updater/env-defaults

```diff
@@ -0,0 +1,48 @@
+# Port the health check server listens on
```

Comment:
How is it getting the Database URL?

Reply:
This is really confusing to me - I'm thinking the env-defaults should include all the environment variables that the app uses. In this case the value needs to be the same for both of the apps otherwise things couldn't work, so I can see how we only have it set in one place.

All that to say is that it seems like there must be a less confusing way for us to define environment variables, not sure what to do.